<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetclint</class>
 <widget class="QWidget" name="widgetclint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pBclose">
   <property name="geometry">
    <rect>
     <x>676</x>
     <y>0</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
image: url(:/images/tb2.png);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tEchat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
    <item>
     <widget class="QComboBox" name="cBsize">
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBjc">
      <property name="toolTip">
       <string>加粗</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images/images.qrc">
        <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBqx">
      <property name="toolTip">
       <string>倾斜</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images/images.qrc">
        <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBxhx">
      <property name="toolTip">
       <string>下划线</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images/images.qrc">
        <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBys">
      <property name="toolTip">
       <string>颜色</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images/images.qrc">
        <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBbc">
      <property name="toolTip">
       <string>保存</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images/images.qrc">
        <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBqc">
      <property name="toolTip">
       <string>清除</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images/images.qrc">
        <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="tEsend">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelnumber">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>在线人数:0人</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tWlist">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>261</width>
     <height>381</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>用户1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>用户2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>用户名:</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>60</number>
    </property>
    <item>
     <widget class="QPushButton" name="pBsend">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBseclet">
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBsendfile">
      <property name="text">
       <string>发送文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
